driver-scala/src/it/scala/org/mongodb/scala/syncadapter/SyncChangeStreamIterable.scala

```diff
@@ -94,4 +94,19 @@ case class SyncChangeStreamIterable[T](wrapped: ChangeStreamObservable[T])
     wrapped.comment(comment)
     this
   }
+
```

Comment:
Please remove the comment, to be consistent with the other methods in this class. (This is a test class, not production).